tests/snapshots.test.js

```diff
@@ -0,0 +1,70 @@
+const {URL} = require('url');
+const snapshotsTest = require('../snapshots');
+
+const PORT = process.env.PORT || 8101;
+
+test('Snapshot widths', async () => {
```

Comment:
I prefer a more descriptive "behavioural" test names that explain what the tested code is expected to do.
This helps in understanding the test code, but also makes sure tests are written from perspective of expected results.

In here I would say something like:
`test("returns correct widths for snapshots, including additional one for responsive examples", … )`
or something like that.